scrunch/datasets.py

```diff
@@ -3508,4 +3515,3 @@ def execute(self, csv_file):
                     folders_by_name[folder_name].entity.delete()
                 # Always delete the tmp dataset no matter what
                 tmp_ds.delete()
```

Comment:
If we delete `tmp_project` here we run into the issue where `DELETE /projecsts/<project_id>` moves expired datasets to the personal folder. @jjdelc what do you think we should do here?